docs/inference_api.md

```diff
@@ -280,18 +280,20 @@ For each loaded model, user can make REST call to URI: /predictions/{model_name}
 ```bash
 curl -O https://s3.amazonaws.com/model-server/inputs/kitten.jpg
 
-curl -X POST http://localhost:8080/predictions/resnet-18 -F "data=@kitten.jpg"
+curl -X POST http://localhost:8080/predictions/resnet-18 -T kitten.jpg
 
 or:
 
-curl -X POST http://localhost:8080/predictions/resnet-18 -T kitten.jpg
+curl -X POST http://localhost:8080/predictions/resnet-18 -F "data=@kitten.jpg"
 ```
 
 The result was some JSON that told us our image likely held a tabby cat. The highest prediction was:
 
 ```json
-"class": "n02123045 tabby, tabby cat",
-"probability": 0.42514491081237793
+{
```

Comment:
Probably some "..." could be used to specify that this response would be larger...
{
"class": ""
"probability" : ""
...
}

Reply:
done